_static/js/custom.js

```diff
@@ -298,6 +297,40 @@ const registerGiscus = function () {
 };
 
 $(document).ready(() => {
+  const httpResponseStatus = window.performance.getEntries()[0].responseStatus;
+  if (httpResponseStatus === 404) {
+    // Check for redirects if on a currently invalid page.
+    // This is done in JavaScript, as Read the Docs' own redirect functionality
+    // is unreliable with large amounts of redirects defined.
+    fetch("/_static/redirects.csv")
```

Comment:
Does this work with the full sub URL too, e.g. `/stable/en`? Normal fetches usually use the base URL, so this would become `docs.godotengine.org/_static/redirects.csv`, which I'm not sure if it would resolve.

Reply:
Fixed (I had this in my local copy, but forgot to copy it out of `_build/html/`).